### PR TITLE
[common-artifact] Support on ubuntu 20.04

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit update common-artifact to support OS on python's default version is greater than 3.8.

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Draft: #9245

### CODE REVIEW

File: compiler/common-artifacts/CMakeLists.txt [hunk 1]
```diff
@@ -33,13 +33,18 @@ set(REQUIREMENTS_FILE "requirements.txt")
 set(REQUIREMENTS_OVERLAY_PATH_TF_2_6_0 "${VIRTUALENV_OVERLAY_TF_2_6_0}/${REQUIREMENTS_FILE}")
 set(REQUIREMENTS_OVERLAY_PATH_TF_2_8_0 "${VIRTUALENV_OVERLAY_TF_2_8_0}/${REQUIREMENTS_FILE}")
 
+set(PYTHON_OVERLAY python3)
```
Comment (mhs4670go) [inline compiler/common-artifacts/CMakeLists.txt:36]:
```cmake
find_package(PythonInterp 3.8 QUIET)
find_package(PythonLibs 3.8 QUIET)
```
Is it okay not to change above line?

File: compiler/common-artifacts/CMakeLists.txt [hunk 2]
```diff
@@ -33,13 +33,18 @@ set(REQUIREMENTS_FILE "requirements.txt")
...
+if(PYTHON_EXECUTABLE MATCHES python3.8)
```
Comment (seanshpark) [inline compiler/common-artifacts/CMakeLists.txt:37]: Ubuntu 18.04 default is python3.6 but we need to use python3.8.
Have you checked this with Ubuntu 18.04 with python3.8 installed but default is python3.8?

File: compiler/common-artifacts/CMakeLists.txt (hunk 1 above)
Comment (hseok-oh) [reply compiler/common-artifacts/CMakeLists.txt:36]: Above line needs to find python version >= 3.8.

File: compiler/common-artifacts/CMakeLists.txt (hunk 2 above)
Comment (hseok-oh) [reply compiler/common-artifacts/CMakeLists.txt:37]: Maybe it's OK because ubuntu 20.04's default python3 is 3.8. I'll check.

Comment (seanshpark) [reply compiler/common-artifacts/CMakeLists.txt:37]: there was a typo and fixed to: is ~python3.8~ python3.6?

Comment (hseok-oh) [reply compiler/common-artifacts/CMakeLists.txt:37]: > Have you checked this with Ubuntu 18.04 with python3.8 installed but default is python3.8 python3.6?

I tested on docker `nnfw/one-devtools:bionic` and its default python is 3.6. It's OK.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**